### PR TITLE
feature/contact-form-visuals

### PR DESCRIPTION
**Pull request by jusk02**
- Contact form layout implemented
- Styles set according to design
- Minor fixes to general input styles to ensure correct spacing

image

image

### CODE REVIEW

File: pages/contact.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <nav id="main-nav" class="main-nav">
       <div class="container main-nav__container">
         <div id="main-nav__logo" class="main-nav__logo">
-          <img class="main-nav__logo-image" src="./../assets/images/logo-hdlp.svg" alt="Hostal Donde Las Paisas">
+            <img class="main-nav__logo-image" src="./../assets/images/logo-hdlp.svg" alt="Hostal Donde Las Paisas">
```
Comment (camigomezdev) [inline pages/contact.html]: Please delete the extra space :)

**Comment by camigomezdev** [comment]
Screen Shot 2019-08-13 at 2 00 16 AM

I think this is a bug, what should we do?

**Comment by jusk02** [comment]
> Screen Shot 2019-08-13 at 2 00 16 AM
> 
> I think this is a bug, what should we do?

Fixed, thanks for that catch

**Review by camigomezdev — APPROVED** [review]
Great! 🎉

**Comment by Frankruiz90** [comment]
image


the phone field lets you enter characters that are not numbers

**Comment by jusk02** [comment]
> image
> 
> the phone field lets you enter characters that are not numbers

@Frankruiz90 
No validations are new, i only took the ones that were available and set them in a independent file for ease of use.

@Frankruiz90 is working on a phone-field validation that will not allow letters and doesnt require the field type to be "number"

**Review by Frankruiz90 — APPROVED** [review]

**Pull request merged**